arch/x86/x86_64/include/uk/asm/compiler.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) YYYY, Copyright Holder. All rights reserved.
```

Comment:
Proper date + name

Reply:
I think this is like this because the files are empty and the person that will fill in code will write themselves there.

Reply:
@razvand, what Cezar said. These are empty, so I'm not sure if it makes any sense to copyright to myself.